governance/votes/Executive vote - April 23, 2021.md

```diff
@@ -0,0 +1,178 @@
+   * [Price Change Interval (`step`)](https://community-development.makerdao.com/en/learn/governance/param-auction-price-function): 90 seconds
+* [Auction Price Multiplier (`buf`)](https://community-development.makerdao.com/en/learn/governance/param-auction-price-multiplier): 1.3
+
+**Limits**
+* [Local Liquidation Limit (`ilk.hole`)](https://community-development.makerdao.com/en/learn/governance/param-local-liquidation-limit): 5 million DAI
+* [Maximum Auction Drawdown (`cusp`)](https://community-development.makerdao.com/en/learn/governance/param-max-auction-drawdown): 0.4
+* [Maximum Auction Duration (`tail`)](https://community-development.makerdao.com/en/learn/governance/param-max-auction-duration): 8,400 seconds
+* [Breaker Price Tolerance (`tolerance`)](https://community-development.makerdao.com/en/learn/governance/param-breaker-price-tolerance): 0.5
+
+**Incentives**
+* [Proportional Kick Incentive (`chip`)](https://community-development.makerdao.com/en/learn/governance/param-proportional-kick-incentive): 0.1%
+* [Flat Kick Incentive (`tip`)](https://community-development.makerdao.com/en/learn/governance/param-flat-kick-incentive): 0 DAI
+
+Further details and rationale can be found in the following [Risk Evaluation](https://forum.makerdao.com/t/yfi-a-liquidations-2-0-parameters/7502).
+
+### Parameter Changes
+
+Given the successful outcome of this [poll](https://vote.makerdao.com/polling/QmRRzmnF?network=mainnet#poll-detail), the following changes on [Stability Fee](https://community-development.makerdao.com/en/learn/governance/param-stability-fee), [Debt Ceiling](https://community-development.makerdao.com/en/learn/governance/param-debt-ceiling/), [Maximum Debt Ceiling, Target Available Debt and Ceiling Increase Cooldown](https://community-development.makerdao.com/en/learn/governance/module-dciam/) will be made if this executive proposal passes.
+
+### Parameter Changes by Type
+
+**Stability Fee Increases**
+
+| Vault Type     |   Old |    New | Change |
+|----------------|-------|--------|--------|
+| LINK-A         | 3.50% | **5.00%** |  1.50% |
+| ETH-B          | 9.00% | **10.00%**|  1.00% |
+| ZRX-A          | 2.00% | **4.00%** |  2.00% |
+| LRC-A          | 3.00% | **4.00%** |  1.00% |
+| UNIV2DAIETH-A  | 3.00% | **3.50%** |  0.50% |
+| UNIV2USDCETH-A | 3.50% | **4.50%** |  1.00% |
+
+**Stability Fee Decreases**
+
+| Vault Type     |   Old |   New | Change |
+|----------------|-------|-------|--------|
+| AAVE-A         | 4.00% | **3.00%** | -1.00% |
+| BAT-A          | 6.00% | **5.00%** | -1.00% |
+| MANA-A         | 4.00% | **3.00%** | -1.00% |
+| BAL-A          | 3.50% | **2.00%** | -1.50% |
+| UNIV2DAIUSDC-A | 3.00% | **1.00%** | -2.00% |
+| UNIV2LINKETH-A | 5.00% | **4.00%** | -1.00% |
+| UNIV2WBTCDAI-A | 2.00% | **0.00%** | -2.00% |
+| UNIV2AAVEETH-A | 5.00% | **4.00%** | -1.00% |
+| UNIV2DAIUSDT-A | 4.00% | **3.00%** | -1.00% |
+
+**Debt Ceiling Increases**
```

Comment:
We should fix titles here so that it's clear this refers to the 'Maximum Debt Ceiling'

Reply:
Changed to "Maximum Debt Ceiling".